### PR TITLE
doc: Use new Sphinx extension with the Getting Started doc

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

File: doc/getting_started/getting_started.rst [hunk 1]
```diff
@@ -81,33 +81,22 @@ To build an example application follow these steps:
 
       $ source zephyr-env.sh
 
-#. Generate project files for the :ref:`hello_world` example, enter:
+#. Build the :ref:`hello_world` example for the `arduino_due` board, enter:
```
Comment (SebastianBoe) [inline doc/getting_started/getting_started.rst]: Says "due" here, and then 101 below?

**Review by SebastianBoe — CHANGES_REQUESTED** [review]

File: doc/getting_started/getting_started.rst (hunk 1 above)
Comment (carlescufi) [reply doc/getting_started/getting_started.rst]: thanks, fixed

**Review by SebastianBoe — APPROVED** [review]

File: doc/getting_started/installation_linux.rst [hunk 2]
```diff
@@ -72,13 +72,13 @@ Install additional packages required for development with Zephyr::
 Note that CMake version 3.8.2+ or higher is required. If you are using a
 distribution that currently ships an older version, check the backports or
 install a more recent version manually. For example, to install version
-3.8.2 from the CMake website directly::
+3.8.2 from the CMake website directly in ~/cmake::
```
Comment (mbolivar) [inline doc/getting_started/installation_linux.rst]: This could be its own patch

**Review by mbolivar — APPROVED** [review]
Looks good, one minor comment
